app/ide-desktop/lib/dashboard/src/layouts/PageSwitcher.tsx

```diff
@@ -1,21 +1,21 @@
 /** @file Switcher to choose the currently visible full-screen page. */
 import * as React from 'react'
 
-import * as tailwindMerge from 'tailwind-merge'
-
 import DriveIcon from 'enso-assets/drive.svg'
-import NetworkIcon from 'enso-assets/network.svg'
+import WorkspaceIcon from 'enso-assets/workspace.svg'
```

Comment:
`Tabs` are just a component. it's not a layout. 
Also, I expect better decomposing than just a straightforward single-file pile

Reply:
see above - i prefer avoiding excess abstractions when there's little to no advantage - in this case, having to go through multiple files to figure out what a single UI element is doing.

as for the layout thing, perhaps it's a bad name - this folder is supposed to be for elements that are only ever used once, in contrast to `components/` which are the reusable elements.

note that i don't expect this to be needed anywhere else, as:
- i don't expect this main tab bar to be hidden in favor of some other tab bar, and
- it can't really be stacked vertically as there is a dark section which would not look good when stacked

Reply:
>  this folder is supposed to be for elements that are only ever used once

Makes sense.
I suppose it's better to put them under a `./components` folder near the usage. 
If a component is a part of a particular route, I see this structure as reasonable:

https://excalidraw.com/#json=PYUh616HdnSaMUuPycZQ6,G5MX86Gs0P5x2xVP1e9m2A

The main purpose of this is the ability to easily "see" the scope of a particular component (its usage scoped to parent element) and colocation.

Reply:
> not a directive to rewrite everything from scratch in this PR

lol, yeah. i was considering maybe moving `PageSwitcher` to this folder structure, but on second thought i think it makes sense to refactor an entire slice/page at a time, so i think it makes sense to do that in a separate PR. thoughts?

Reply:
Sure, I'd say it's a good start to write new code with the new structure in mind. We can start with next changes and eventually refactor by following the boy scout rule